pdf/config.json5

```diff
@@ -144,6 +144,80 @@
                 '-e', '<inputFile>'],
             version: 'pdfid_v0_2_7__py3',
             parse: {
+                /** Parser output post-processing specification for type 'regex-counter'
```

Comment:
@wwoods Here's the documentation of the `regex-counter` parser configuration. You probably don't need to read the rest of the PR, but please let me know if you spot any issues with this section.